C/c4Document.cc

```diff
@@ -39,17 +39,15 @@ using namespace litecore;
 
 
 C4Document::C4Document(C4Collection *collection, alloc_slice docID_)
-:_collection(asInternal(collection))
-,_docID(move(docID_))
-{
+        : _collection(asInternal(collection)), _docID(move(docID_)) {
```

Comment:
Why are the initializers indented way over like this?

Reply:
Fixed with move to Clang-Format